VerticalStaggeredVolumes.m

```diff
@@ -1,36 +1,39 @@
 classdef VerticalStaggeredVolumes < StaggeredVolumes
 	properties
 		uidx; % 4xNV
-		% 4 1
-		% 3 2
+		% 3 1
+		% 4 2
 		vidx; % 5xNV
 		%   2
-		% 5 1 3
-		%   4
+		% 5 1 4
+		%   3
 	end
 
 	methods
 		function this = VerticalStaggeredVolumes(mesh)
-			this.vidx = this.getNeighbors(mesh);
+			this.vidx = [ 1:mesh.NV      ;
+						  mesh.rel(1, :) ;
+						  mesh.rel(3, :) ;
+						  mesh.rel(2, :) ;
+						  mesh.rel(4, :) ];
 			
 			this.uidx = [ mesh.rel(1, :)               ;
 			              1:mesh.NV                    ;
-			              mesh.rel(4, :)               ;
-			              mesh.rel(1, mesh.rel(4, :)) ];
+			              mesh.rel(1, mesh.rel(4, :))  ;
+						  mesh.rel(4, :)               ];
 		end
 
 		function c = convective(this, mesh, uv)
```

Comment:
These functions can be moved to the base class, and simply change the order of parameters that are passed to it, in the way the diffusive term is calculated.

Reply:
You are right about the uidx function, I'll recover the getNeighbors method.

About the function to unify convective term methods, I thought about it but maybe we are adding too many function calls? (it's still MATLAB 😂)